packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../variables';
+
+:local {
+  .logo-container {
+    margin: 30px 25px;
+    max-height: 170px;
+    overflow: hidden;
+    width: 270px;
```

Comment:
i feel like this should be fluid... set to 100% so it grows/changes if the nav width changes